Manito/Manito/Screens/Detail-Wait/ViewModel/DetailEditViewModel.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  DetailEditViewModel.swift
+//  Manito
+//
+//  Created by Mingwan Choi on 10/31/23.
+//
+
+import Combine
+import Foundation
+
+final class DetailEditViewModel: BaseViewModelType {
+    struct Input {
+        let viewDidLoad: AnyPublisher<Void, Never>
+        let changeButtonDidTap: AnyPublisher<CreatedRoomInfoRequestDTO, Never>
+    }
+    struct Output {
+        let roomInfo: AnyPublisher<RoomInfo, Never>
+        let passStartDate: AnyPublisher<Bool, Never>
+        let overMember: AnyPublisher<Bool, Never>
```

Comment:
p5;

변수명이 저는 is가 앞에 붙으면 좋겠어요!

```suggestion
        let isPassStartDate: AnyPublisher<Bool, Never>
        let isOverMember: AnyPublisher<Bool, Never>
```

Reply:
넵넵! is를 붙이니 더 명확해지는군요!
감사합니다 😊
[e4cac67](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/600/commits/e4cac673ec3ac8d6a1967c2dd3d2e49e20bdec44)